entries/entry02.md

```diff
@@ -1,7 +1,28 @@
 # Entry 2
```

Comment:
Sources: 1
EDP: 0
Knowledge: 1
Skills: 0
Professionalism: 0 (many typos and spacing errors).

Integrate sources into your writing (not just linked at bottom).